server/bin/pbench-unpack-tarballs.sh

```diff
@@ -37,6 +37,21 @@
 # symlink from $ARCHIVE/TODO to $ARCHIVE/TO-COPY-SOS.
 
 
+BUCKET="${1}"
+PIPELINE="${2}"
+
+if [[ "${PIPELINE}" == "re-unpack" ]]; then
+    export PROG="pbench-re-unpack-tarballs"
+fi
+if [[ -z "${BUCKET}" || "${BUCKET}" == "none" ]]; then
```

Comment:
Rather than doing this compound test (which we do twice, now -- see line 68), how about handling it at line 40:
```
BUCKET=${1:-none}
```
Then lines 40 and 68 are reduced to just
```
if [[ ${BUCKET} == "none" ]]; then
```

Reply:
line 46* and 68